tests/options/pugExplicitDiv/pug-explicit-div.test.ts

```diff
@@ -0,0 +1,48 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { format } from 'prettier';
+import { plugin } from '../../../src/index';
+
+describe('Options', () => {
+	describe('pugExplicitDiv', () => {
+		const code: string = readFileSync(resolve(__dirname, 'unformatted.pug'), 'utf8');
+		const commonOpts = {
+			parser: 'pug',
+			plugins: [plugin],
+			// use this common options in all tests to force specific wrapping
+			pugAttributeSeparator: 'none',
+			pugPrintWidth: 80,
+		}
+
+		test('should handle unspecified pugExplicitDiv', () => {
+			const expected: string = readFileSync(resolve(__dirname, 'formatted-implicit-div.pug'), 'utf8');
+			const actual: string = format(code, {
+				...commonOpts,
+			});
+
+			expect(actual).toBe(expected);
+		});
+
+		test('should handle pugExplicitDiv:false', () => {
+			const expected: string = readFileSync(resolve(__dirname, 'formatted-implicit-div.pug'), 'utf8');
+			const actual: string = format(code, {
+				// @ts-expect-error
+				pugExplicitDiv: false,
```

Comment:
Explicit "override" values should be placed after spreading so it will always be taken, also if someone write this value in the common object

Reply:
oops, yes - will fix.